mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDaoImpl.java

```diff
@@ -137,6 +136,10 @@ public class JournalRecordDaoImpl implements JournalRecordDao {
   private static final Logger LOGGER = LogManager.getLogger();
   public static final String SOURCE_RECORD_ENTITY_TYPE = "source_record_entity_type";
   public static final String ORDER_ENTITY_ID = "order_entity_id";
+  public static final String SOURCE_ENTITY_ERROR = "source_entity_error";
```

Comment:
looks like `HOLDINGS_ENTITY_HRID` and `SOURCE_ENTITY_ERROR` constants are already defined in `JournalRecordsColumns`